website/content/tutorials/1-getting-started/_index.md

```diff
@@ -0,0 +1,954 @@
+      <architecture>x86-64</architecture>
+      <speed>4.7 gigahertz</speed>
+    </cpu>
+    <ata-socket>
+      <product-name>ISA Corp SuperSATA Model 2 Storage Socket</product-name>
+      <type>sata</type>
+    </ata-socket>
+    <memory>
+      <product-name>Massive Memory Corp Model 3 DDR4-3200 8GB (Module 1)</product-name>
+      <byte-size>8589934592</byte-size>
+    </memory>
+    <memory>
+      <product-name>Massive Memory Corp Model 3 DDR4-3200 8GB (Module 2)</product-name>
+      <byte-size>8589934592</byte-size>
+    </memory>
+  </motherboard>
+</computer>
+{{< /highlight >}}
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml >}}
+---
+computer:
+  id: awesomepc1
+  vendor:
+    id: vendor1
+    name: AwesomeComp Incorportated
+    address: 1000 K Street NW Washington, DC 20001
+    website: https://example.com/awesomecomp/awesomepc1/details
+  motherboard:
+    product-name: ISA Corp Magestic Model M-Ultra Motherboard
+    type: atx
+    cpu:
+      product-name: ISA Corp Superchip Model 1 4.7 GHz 4-core Processor
+      architecture: x86-64
+      speed: 4.7 gigahertz
+    ata-socket:
+      product-name: ISA Corp SuperATA Model 2 Socket for SATA Drive
+      type: sata
+    memory-modules:
+    - product-name: Massive Memory Corp Model 3 8GB DDR4-3200 (Module 1)
+      size: 8589934592
+    - product-name: Massive Memory Corp Model 3 8GB DDR4-3200 (Module 2)
+      size: 8589934592
+{{< /highlight >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+With the expressive power of assemblies, flags, and fields, we can specify complex managed objects and control the structure of the intended information model in the resulting data formats.
+
+Our Metaschema-enabled tools can parse and generate the different data formats. We specify flags on the `computer` assembly, but all else we define in the `model` of the `assembly`. And within that model, we can define the motherboard `assembly` inline with its own `flag` and `model`. XML has different syntax and semantics from JSON and YAML that would lead to structural limitations we would need to work around ourselves by manually designing separate data formats without Metaschema. With Metaschema, we can use `group-as` declarations  in our definition, and Metaschema-enabled tools will generate and parse instances for us as instances of the same, despite these limitations, in different data formats automatically.
+
+We define the data types for different Metaschema fields and flags. Our Metaschema-enabled tools can leverage pre-compiled schemas or generate their own to enforce `field` and `flag` values that are valid for their type. For example, our Metaschema-enabled tools should accept a valid URI for the `website` field of the vendor `assembly`, but not any arbitrary string. For `byte-size`, they should only accept positive integer values greater than 0, not a decimal point number or string. Metaschema facilitates consistent enforcement of data typing so we developers do not have to.
```

Comment:
It is not worth remarking that these are lexical data types bound to string values stored in leaf nodes (terminal nodes) in the tree, is it?

Reply:
This is a good point I think we need to make, but perhaps in a subsequent tutorial that is more advanced? I do not want to flat out dismiss that, but later we approach the discussion of tree structure and how it relates, perhaps when discussing indices? (Because then we have to get into Metapath using XPath-like `.` as the short-form `current()` context operator and starts becoming more relevant right?

We also have an allowed-values tutorial asked for next, so maybe that will come up there, or as ancillary thing about how Metapath works. Thanks for this feedback, I had not even thought about that!

Reply:
Noted it in https://github.com/usnistgov/metaschema/issues/292#issuecomment-1379126937.